testsuite/features/init_clients/srv_create_activationkey.feature

```diff
@@ -9,13 +9,28 @@ Feature: Create activation keys
   Scenario: Log in as admin user
     Given I am authorized for the "Admin" section
 
+  Scenario: Clone the child custom channel including test repositories
+    When I follow the left menu "Software > Manage > Channels"
+    And I follow "Clone Channel"
+    And I select the custom architecture channel for "sle_minion" as the origin channel
+    And I choose "current"
+    And I click on "Clone Channel"
+    And I enter "SLE-Custom-Channel-x86_64" as "Channel Name"
```

Comment:
I think we still should include the word "test" in this custom channel.
Custom channel is the kind of channel, test channel describes the purpose of this channel. Will "SLE-Test-Custom-Channel-x86_64" work for you?